misc/acrn-config/config_app/requirements

```diff
@@ -1,3 +1,4 @@
 Flask==1.1.1
 flask_bootstrap==3.3.7.1
-
+xmlschema==1.3.1
```

Comment:
The latest xmlschema library is 1.4.1. Shall we use that.

Reply:
Sure. I will update it to version 1.4.1